### ISSUE TITLE
FR: Fix model for Linux

### ISSUE THREAD

**Issue by tim-gromeyer** [bambulab/BambuStudio#4329]
**Is your feature request related to a problem? Please describe.**
Hello, I would like to use the "Fix model" option on Linux.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
I saw, that this functionality is implemented using the windows sdk (`src/slic3r/Utils/FixModelByWin10.cpp`)

**Comment by alanLacey44** [comment]
Yes, me too.  Don't like having to go to Windows for non-manifold fixes.  Sadly I can't install the studio exe under Wine in my Mint 21.1

**Comment by freckletonj** [comment]
same, this would be great!

**Comment by louhy** [comment]
"Use Windows" is a bit of an infuriating suggestion. I'm not installing another OS just for this, and certainly not paying for it either. We need this on Linux. Or at least have it tell us where the problem is!

**Comment by louhy** [comment]
Why is this OS specific, anyway? What's it doing that isn't portable?

**Comment by tim-gromeyer** [comment]
It's using the Windows API to repair the model.

**Comment by louhy** [comment]
...something.. built into Windows? _Weird_ (no wonder it's so bloated :stuck_out_tongue_closed_eyes:)
